APISpecification.yaml

```diff
@@ -475,3 +532,84 @@ components:
         - related_genomes_count
       additionalProperties: false
       type: object
+    Karyotype:
+      type: array
+      items:
+        $ref: '#/components/schemas/TopLevelRegion'
+    TopLevelRegion:
+      type: object
+      properties:
+        name:
+          type: string
+          example: "1"
+        type:
+          type: string
+          example: "chromosome"
+        length:
+          type: integer
+          example: 248956422
+        is_circular:
+          type: boolean
+          example: false
+    RegionValidationResponse:
+      type: object
+      properties:
+        genome_id:
```

Comment:
I've copied genome_id field from the api spec in ensembl-2020-genome-search; but I cannot remember why we included it in the region validation response in the first place. It is, in theory, possible to send a wrong genome id in the request; but web client shouldn't ever do it.